src/app/aliases/aliases.component.ts

```diff
@@ -17,12 +17,14 @@ export class AliasesComponent extends Base implements OnInit {
   loading$: Observable<boolean>;
   organization$: Observable<Organization>;
   collection$: Observable<Collection>;
+  workflow$: Observable<Workflow>;
+  tool$: Observable<DockstoreTool>;
 
   public type: string;
   public alias: string;
   public validType: boolean;
   // Types contains resource types that support aliases
-  public types = [ 'organizations', 'collections' ];
+  public types = [ 'organizations', 'collections', 'workflows', 'tools', 'containers' ];
```

Comment:
Just curious, how can the type of a tool be either tools or containers? Is this some legacy thing?

Reply:
Yes, we called tools containers a long time ago, so we don't want to break links.